src/MongoDB.Driver/MongoDB.Driver.csproj

```diff
@@ -42,7 +42,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2" PrivateAssets="All" />
-    <PackageReference Include="MongoDB.Libmongocrypt" Version="1.0.0" />
+    <PackageReference Include="MongoDB.Libmongocrypt" Version="0.0.0-local" />
```

Comment:
I assume you will fix this before pushing to master.

Reply:
Correct